src/main/java/pl/isa/biblioteka/AddUsers.java

```diff
@@ -1,33 +1,39 @@
 package pl.isa.biblioteka;
 
 import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 
 public class AddUsers {
     //Mikołaj Malinowski
     //Dodanie nowego użytkownika
-    ArrayList<String> users = new ArrayList<>();
+    public static Person person;
+    public static List<Person> users = new ArrayList<>();
 
     public void addUser() {
-        String firstName;
-        String lastName;
-        String name;
-
         Scanner input = new Scanner(System.in);
         System.out.println("Podaj imię nowego użytkownika: ");
-        firstName = input.nextLine();
+        String firstName = input.nextLine().trim();
         System.out.println("Podaj nazwisko nowego użytkownika: ");
-        lastName = input.nextLine();
-        name = firstName + " " + lastName;
-        users.add(name);
-        System.out.println("Dodano nowego użytkownika: " + name);
+        String lastName = input.nextLine().trim();
+        Person addPerson = new Person(firstName, lastName);
+        users.add(addPerson);
+        System.out.printf("Dodano nowego użytkownika: %s %s ", firstName, lastName);
     }
 
-    public void listUsers() {
-        System.out.println("Użytkownicy: ");
-        for (int i = 0; i < users.size(); i++) {
-            System.out.println(i + 1 + " " + users.get(i));
+    public void listsUsers(){
+        System.out.println("Uzytkownicy w naszej aplikacji: ");
+        int index = 1;
```

Comment:
Ten index jest do czegoś potrzebny?

Reply:
ogólnie wykorzystałem go do tego aby wyświetlić numer kolejnego użytkownika przy wyświetlaniu listy. Może to zostać czy raczej się tak nie robi?